src/app/containers/Article/index.jsx

```diff
@@ -11,50 +11,79 @@ import ArticleMain from '../ArticleMain';
 import ErrorMain from '../ErrorMain';
 import nodeLogger from '../../helpers/logger.node';
 import ConsentBanner from '../ConsentBanner';
+import { GhostWrapper, GridItemConstrainedMedium } from '../../lib/styledGrid';
 
 const logger = nodeLogger(__filename);
 
-/*
-  [1] This handles async data fetching, and a 'loading state', which we should look to handle more intelligently.
-*/
-const ArticleContainer = ({ loading, error, data, bbcOrigin }) => {
-  if (loading) return 'Loading...'; /* [1] */
-  if (error) {
-    logger.error(error);
-    return 'Something went wrong :(';
-  }
+const ArticleWrapper = ({ bbcOrigin, children, isAmp, service }) => (
+  <Fragment>
+    <GlobalStyle />
+    <RequestContextProvider
+      platform={isAmp ? 'amp' : 'canonical'}
+      bbcOrigin={bbcOrigin}
+    >
+      <ServiceContextProvider service={service}>
+        <Helmet>
+          <link rel="manifest" href={`/${service}/articles/manifest.json`} />
+        </Helmet>
+        <ConsentBanner />
+        <HeaderContainer />
+        {children}
+      </ServiceContextProvider>
+    </RequestContextProvider>
+  </Fragment>
+);
 
+ArticleWrapper.propTypes = {
+  bbcOrigin: string,
+  children: node.isRequired,
+  isAmp: bool.isRequired,
+  service: string.isRequired,
+};
+
+ArticleWrapper.defaultProps = {
+  bbcOrigin: null,
+};
+
+const ArticleContainer = ({ loading, error, data, bbcOrigin }) => {
   if (data) {
-    const { isAmp, data: articleData, service, status } = data;
+    const { isAmp, service } = data;
+
+    if (loading) {
+      return (
+        <ArticleWrapper isAmp={isAmp} bbcOrigin={bbcOrigin} service={service}>
+          <main role="main">
+            <GhostWrapper>
+              <GridItemConstrainedMedium />
+            </GhostWrapper>
+          </main>
+        </ArticleWrapper>
+      );
+    }
+
+    if (error) {
+      logger.error(error);
+      return (
+        <ArticleWrapper isAmp={isAmp} bbcOrigin={bbcOrigin} service={service}>
+          <ErrorMain status={500} />
+          <FooterContainer />
+        </ArticleWrapper>
+      );
+    }
+
+    const { data: articleData, status } = data;
```

Comment:
You shouldn't need to split the usage out like this, it kinda hints to whats used when, but isn't necessary IMO